.semaphore/semaphore.yml

```diff
@@ -116,10 +116,9 @@ blocks:
           - echo "Installing Performous dependencies..."
           - sudo add-apt-repository -y ppa:litenstein/focal-backports
           - sudo apt-get update -y
-          - sudo apt-get install -y --no-install-recommends ninja-build
-          - sudo apt-get install -y --no-install-recommends gettext clang-12 gcc-11 g++-11 python3-dev python3-pip meson libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-program-options-dev libboost-iostreams-dev libboost-system-dev libboost-locale-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++3.2-dev portaudio19-dev libicu-dev libopencv-dev libportmidi-dev libglm-dev libaubio-dev libcpprest-dev
-          - sem-version python 3.8
-#          - sudo pip3 install meson
+          - sudo apt-get install -y ninja-build
+          - sudo apt-get install -y ninja-build gettext clang-12 gcc-11 g++-11
+          - sudo apt-get install -y libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-iostreams-dev libboost-locale-dev libboost-system-dev libboost-program-options-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++3.2-dev portaudio19-dev libicu-dev libopencv-dev libportmidi-dev libglm-dev libaubio-dev libcpprest-dev meson
```

Comment:
Please use the `--no-install-recommends` flag as this speeds up the build by about 20-30%